docs/develop/typescript-restrictions.md

```diff
@@ -17,9 +17,9 @@ Writing [types](https://www.typescriptlang.org/docs/handbook/typescript-in-5-min
 
 You cannot explicitly declare a variable to be of type `any` in Office Scripts (that is, `let someVariable: any;`). The `any` type causes issues when processed by Excel. For example, a `Range` needs to know that a value is a `string`, `number`, or `boolean`. You will receive a compile-time error (an error prior to running the script) if any variable is explicitly defined as the `any` type in the script.
 
-![The explicit any message in the code editor's hover text](../images/explicit-any-editor-message.png)
+:::image type="content" source="../images/explicit-any-editor-message.png" alt-text="The explicit any message in the code editor's hover text":::
 
-![The explicit any error in the console window](../images/explicit-any-error-message.png)
+:::image type="content" source="../images/explicit-any-error-message.png" alt-text="The explicit 'any' error in the console window":::
 
 In the above screenshot `[5, 16] Explicit Any is not allowed` indicates that line #5, column #16 defines `any` type. This helps you locate the error.
```

Comment:
```suggestion
In the previous screenshot `[5, 16] Explicit Any is not allowed` indicates that line #5, column #16 defines `any` type. This helps you locate the error.
```
For accessibility, use "previous" and "following" instead of "above" and "below."